docs-requirements.txt

```diff
@@ -1,5 +1,6 @@
 docutils>=0.15.2,<0.17
 pydata-sphinx-theme>=0.3.1
+astroid<=2.6.6
```

Comment:
If we're gonna do this in the release PR please add a separate log entry mentioning it